config/sync/core.entity_view_display.node.article.default.yml

```diff
@@ -28,108 +54,93 @@ bundle: article
 mode: default
 content:
   banner_display:
-    weight: 0
+    weight: 1
+    region: content
     settings: {  }
     third_party_settings: {  }
-    region: content
   body:
     type: text_default
-    weight: 4
+    weight: 5
     region: content
     settings: {  }
     third_party_settings: {  }
-    label: above
+    label: hidden
   content_moderation_control:
-    weight: 0
-    region: content
-    settings: {  }
-    third_party_settings: {  }
-  field_additional_info:
-    type: text_default
-    weight: 10
+    weight: 2
     region: content
-    label: above
     settings: {  }
     third_party_settings: {  }
-  field_banner_image:
-    weight: 15
-    label: above
-    settings:
-      link: true
-    third_party_settings: {  }
-    type: entity_reference_label
-    region: content
   field_meta_tags:
-    weight: 13
+    type: metatag_empty_formatter
+    weight: 12
+    region: content
     label: above
     settings: {  }
     third_party_settings: {  }
-    type: metatag_empty_formatter
-    region: content
   field_photo:
-    type: entity_reference_entity_view
-    weight: 14
-    label: above
+    type: media_thumbnail
```

Comment:
@johangant is this undoing the changes you did recently?

Reply:
@omahm it's the default view mode so shouldn't matter. I'd be inclined to move this to a new view mode called 'diff', or something like that for reasons I've outlined in a PR comment below.

Reply:
I added the Photo field into the default display mode for revision comparison purposes.  However the image style should be 'None (original image)' - not media thumbnail.  We'll need to review all image styles and apply them appropriately at a later stage (theres a task in backlog for that).